app/assets/javascripts/common/admins.js

```diff
@@ -1,97 +1,117 @@
+//
```

Comment:
This file has a bunch of refactoring, but the main two optimizations are inline commented down below.